docs/developing.md

```diff
@@ -26,4 +26,7 @@ git push origin vMAJOR.MINOR.PATCH
 
 For cases that do not fit into the above, refer to the [semver spec](semver2) (and consider updating this documentation).
 
+### Prereleases
+If the version is alpha or beta, for example `v0.0.1-alpha.2`, the CI should automatically make the release into a prerelease. All that is required for this to occur is that there is some text after the version `v0.0.1`.
```

Comment:
I think it might be helpful to include a link to [the part of the semver spec concerning prerelease labels](https://semver.org/#spec-item-9) here  to help ensure that prerelease git tags match those rules